### ISSUE TITLE
Long load time on linux

### ISSUE THREAD

**Issue by ashleylester** [SabakiHQ/Sabaki#403]
AppImage on Kubuntu 18.04 takes 30 seconds or over. Can you look into?

    ash@thinkpad:~$ uname -a
    Linux thinkpad 4.15.0-22-generic #24-Ubuntu SMP Wed May 16 12:15:17 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

Please let me know if you need any more information.

**Comment by yishn** [comment]
Can you try [building Sabaki yourself](https://github.com/SabakiHQ/Sabaki/blob/master/docs/guides/building-tests.md) using the `npm run build` command and see if the generated executable takes as long to start?

**Comment by ashleylester** [comment]
I think the electron-builder builds are currently not working on my distro, I get this error: https://github.com/electron-userland/electron-builder/issues/2958

Incidentally sudo apt-get install desktop-file-utils is needed for the AppImage it seems.

**Comment by yishn** [comment]
I think this error only exists when packaging using AppImage. Creating executables with `npm run build` should work without problems.

**Comment by ashleylester** [comment]
Cool. So it opens in less than a second using the executable under npm run build.

**Comment by yishn** [comment]
Hmm... it‘s an AppImage problem, then... as you have noticed, electron-builder seems to have some problems creating the AppImage format at the moment.

**Comment by yishn** [comment]
Does this issue still persist in the latest version v0.35.0?

**Comment by D4id4los** [comment]
I measured 21 seconds startup with the AppImage build of v0.35.0 on Ubuntu 18.04 while the self-built version started in 2.5 seconds. 
> :~$ uname -a
> Linux nexus-ub 4.15.0-29-generic #31-Ubuntu SMP Tue Jul 17 15:39:52 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

**Comment by egri-nagy** [comment]
Yes, it is still there with the AppImage v0.35.1, Ubuntu 18.04. I have similar measurements on several machines.

**Comment by elsantodel90** [comment]
I can confirm this is still an issue in v0.35.1, Ubuntu 16.04, super annoying.

**Comment by apetresc** [comment]
I'm not even certain this is a problem specific to `electron-builder`; Snaps and AppImages have _always_ had obscene startup times for me :(

**Comment by hadim** [comment]
I confirm the issue for me on Ubuntu 18.04.

**Comment by yishn** [comment]
I've uploaded new AppImages for v0.35.1 with @hadim's fix. They're slightly larger, but should resolve the problem.
